core/src/test/resources/flows/valids/task-flow.yaml

```diff
@@ -5,6 +5,9 @@ inputs:
   - name: string
     type: STRING
 
+labels:
+  flowLabel: flowFoo
```

Comment:
Please also add labels to switch flow and within the "launch" subflow task to ensure you're not overriding subflow default labels